engines/private/detection.cpp

```diff
@@ -209,11 +209,11 @@ static const ADGameDescription gameDescriptions[] = {
 	},
 	{
 		"private-eye", // MacOS demo (US) uninstalled
-		_s("Compressed game detected. Please uncompress it as specified in the game description on our Wiki"),
-		AD_ENTRY1s("Private Eye Demo Installer", "e7665ddc5e6d932c4a65598ecc4ec7d2", 1626393),
+		0,
+		AD_ENTRY1s("Private Eye Demo Installer", "fefd0b105e907893de278bb3aad404e3", 107002),
```

Comment:
Why you need to change the hashes?

Reply:
Because hash here is based on the file copied as data-fork-only or AppleDouble. With dumper-companion we will get macbinary here. The best way is to hash the data fork. For this we need https://github.com/scummvm/scummvm/pull/4549 to go in first. Until then hashing resource fork is the best we can do

Reply:
Not the case anymore